config/base.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  root: true,
+
+  parser: 'vue-eslint-parser',
+
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+
+  env: {
+    browser: true,
```

Comment:
I'd keep this simple 'browser' setting for now, at least for v3.0.0-beta to get a better picture about possible issues with this during feedback gathering period. WDYT @mysticatea ?